vite.config.ts

```diff
@@ -43,45 +39,23 @@ export default defineConfig({
     build: {
         lib: {
             entry: resolve(__dirname, "lib/main.ts"),
-            formats: ["es"],
+            fileName: (format) => `main.${format === "es" ? "mjs" : "cjs"}`,
+            formats: ["es", "cjs"],
+            rollupOptions: {
+                output: {
+                    manualChunks(id) {
+                        if (id.includes("node_modules")) {
+                            return "vendor";
+                        }
+                    },
+                },
+            },
```

Comment:
having this error in local
'rollupOptions' does not exist in type 'LibraryOptions'